constellations/gemini-admin/src/emailPayment/accountInfo.js

```diff
@@ -0,0 +1,151 @@
+
+        var emails = [];
+        emails = users.map((user) => user.email);
+
+        const userRows = users.map((user, index) => {
+            var status = user.isGuardian ? "(guardian" : "(student";
+            status += user.email == accountEmail ? ", primary contact)" : ")";
+
+            if (this.state.selectedUsers.length > 0) {
+                for (var i = 0; i < this.state.selectedUsers.length; i++) {
+                    if (
+                        users.find(
+                            (user) => user == this.state.selectedUsers[i]
+                        ) == undefined
+                    ) {
+                        this.setState({
+                            selectedUserEmails: [],
+                            selectedUsers: [],
+                        });
+                        break;
+                    }
+                }
+            }
+
+            return (
+                <div className="row" key={index}>
+                    <input
+                        type="checkbox"
+                        onChange={(e) => this.onCheckUser(e, user.id)}
+                    />
+                    <span className="column">{getFullName(user)}</span>
+                    <span className="column">{user.email}</span>
+                    <span className="column status">{status}</span>
+                </div>
+            );
+        });
+
+        var balance = 0;
+        transactions.map((transaction, index) => {
+            const amount = transaction.amount;
+            balance += parseInt(amount);
+        });
+
+        return (
+            <section id="account-info">
+                <section id="account-information">
+                    <span id="account-number">Account Information</span>
+                    <h2>AccountId: {id}</h2>
+                    <h2>Primary Email: {accountEmail}</h2>
+                    <div id="account-users">
+                        <h2>Users in Account</h2>
+                        {userRows}
+                    </div>
+                    <div id="transaction-footer">
+                        <span>
+                            Account Balance:{" "}
+                            <b>{this.formatCurrency(balance)}</b>
+                        </span>
+                    </div>
+                </section>
+
+                <section id="generated-email">
+                    <div className="email-template">
+                        <span id="template-title">
+                            Generated Email Template
+                        </span>
+                        <h3>To: {this.state.selectedUserEmails.toString()}</h3>
+                        <h3>
+                            Subject: Math Navigator: Account Balance Payment
+                            Reminder
+                        </h3>
+                        <h3>Message: </h3>
+                        <div className="generated-email">
+                            <p>Hello {name},</p>
+                            <p>
```

Comment:
Can you add a line here in the email that says your current account balance is $____
which should be the `balance` variable you calculated above.